src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3978,20 +3978,54 @@ impl Coordinator {
 
     pub(super) async fn sequence_alter_role(
         &mut self,
-        session: &Session,
-        plan::AlterRolePlan {
-            id,
-            name,
-            attributes,
-        }: plan::AlterRolePlan,
+        session: &mut Session,
+        plan::AlterRolePlan { id, name, option }: plan::AlterRolePlan,
     ) -> Result<ExecuteResponse, AdapterError> {
         let catalog = self.catalog().for_session(session);
         let role = catalog.get_role(&id);
-        let attributes = (role, attributes).into();
+
+        // Get the attributes and variables from the role, as they currently are.
+        let mut attributes = RoleAttributes::new();
+        attributes.inherit = role.is_inherit();
```

Comment:
Why not just do something like `let attributes = role.attributes.clone()`? I worry this is an easy thing to not update if we add more attributes.

Reply:
Unforunately `role` here is a `&dyn CatalogRole`, which doesn't expose attributes. I wasn't sure if this was on purpose or not so I didn't add it.

For now I'll add it because I agree `role.attributes.clone()` is a much better approach, and I can remove it we shouldn't have that method on `trait CatalogRole`.